commune_app/templates/commune_app/chores.html

```diff
@@ -0,0 +1,61 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Communet</title>
+    <link rel="stylesheet" href="{% static 'css/style.css' %}"> 
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js" integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.min.js" integrity="sha384-cuYeSxntonz0PPNlHhBs68uyIAVpIIOZZ5JqeqvYYIcEL727kskC66kF92t6Xl2V" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"/>
+</head>
+<body>
+    <!--NAVIGATION BAR-->
+    <nav class="navbar sticky-top navbar-light navbar-expand-lg">
+        <div class="container max-width">
+        <a href="#" class="navbar-brand">
+        <img class="logo" src="{% static 'images/Communet.png' %}" alt="Logo" title="Logo"></a>
+        <button class="navbar-toggler bg-light" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-label="navbar">
+        <span class="navbar-toggler-icon bg-light">
+        </span></button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+            <li class="nav-item"><a href="{% url 'Main_Page' %}" class="nav-link active text-white">Home</a></li>
+            <li class="nav-item"><a href="{% url 'Main_Page' %}" class="nav-link active text-white">About Us</a></li>
+            <li class="nav-item"><a href="{% url 'chores' %}" class="nav-link active text-white">Chores</a></li>
+            <li class="nav-item"><a href="{% url 'Main_Page' %}" class="nav-link active text-white">Team</a></li></ul>
+        </div>
+        </div>
+    </nav>
+    <main>
+        <div class="container-fluid text-center p-5">
+            <h3>CHORES</h3>
+        </div>
+        <div class="container">
+            <div class="row">
+                {% block content %}
+                    {% for i in Chores %}
```

Comment:
it is more readable like:
```suggestion
                    {% for chore  in Chores %}
```

Reply:
done fixed.